src/pages/Main/TimeSale/TimeSale.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect } from 'react';
+import './TimeSale.scss';
+
+import time from './Icon/time.svg';
+
+export default function TimeSale({ timesale }) {
+  const DAY_HOUR = 24;
+  const DAY_MINUTE = 59;
+  const DAY_SECOND = 59;
+  const [hour, setHour] = useState(DAY_HOUR - new Date().getHours());
+  const [minute, setMinute] = useState(DAY_MINUTE - new Date().getMinutes());
+  const [second, setSecond] = useState(DAY_SECOND - new Date().getSeconds());
+
+  useEffect(() => {
+    const id = setInterval(() => {
+      setHour(23 - new Date().getHours());
+      setMinute(59 - new Date().getMinutes());
+      setSecond(59 - new Date().getSeconds());
```

Comment:
```suggestion
      setHour(23 - new Date().getHours());
      setMinute(59 - new Date().getMinutes());
      setSecond(59 - new Date().getSeconds());
```

이부분의 23,59도 변수로 처리 해주시는게 좋겠습니다

Reply:
반영이 안됐네요